README.md

```diff
@@ -16,7 +16,7 @@ Run [Django](https://www.djangoproject.com) projects from source using [Gunicorn
 #### Step 1: Get your Django project in shape
 There are a few ways that your Django project needs to be set up in order to be compatible with this Docker image.
 
-**setup.py**  
+**setup.py**
```

Comment:
These double-spaces are newlines in markdown.

Reply:
gah my editor seems to remove them automatically